tests/gold_tests/basic/copy_config.test.py

```diff
@@ -37,14 +38,14 @@
 t = Test.AddTestRun("Talk to ts1")
 t.Processes.Default.StartBefore(ts1)
 t.Processes.Default.StartBefore(ts2)
-t.MakeCurlCommand("127.0.0.1:{port}".format(port=ts1.Variables.port))
+t.MakeCurlCommand("127.0.0.1:{port}".format(port=ts1.Variables.port), uds_path=ts1.Variables.uds_path)
```

Comment:
I suggest passing ts rather than uds_path for each of these. That way the user doesn't need to think of the uds_path for each of these tests (it's easier just to pass ts) and if something else like this is overlaid in MakeCurlCommand based on ts it can grab it from there.